### ISSUE TITLE
improve native effect summary in VttCue.line in dart:html

### ISSUE THREAD

**Issue by sigmundch** [dart-lang/sdk#26029]
Compiling a large app I noticed that all native elements were retained due to `VttCue.line`.

@jacob314

**Comment by alan-knight** [comment]
What do you mean by native effect summary? It has an "@Returns('num|String')". Does it also need a Creates annotation? Or is there something else it needs?

**Comment by sigmundch** [comment]
Yeah, I believe it need a `@creates` annotation too, probably `@Creates("Null")` is good enough (since num/String are everywhere.

I just noticed because I was doing something unrelated and run dart2js with `--verbose`. It mentioned that this was making us retain all native elements. It might be that after fixing this others will come up though.
